common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -134,6 +135,29 @@ public TachyonConf() {
       throw new RuntimeException("Unable to load default Tachyon properties file.", e);
     }
 
+    // Override default properties if there exist corresponding environment variables
+    if (getenv("TACHYON_MASTER_ADDRESS") != null) {
+      defaultProps.setProperty(Constants.MASTER_HOSTNAME, getenv("TACHYON_MASTER_ADDRESS"));
+    }
+    if (getenv("TACHYON_MASTER_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.MASTER_BIND_HOST, getenv("TACHYON_MASTER_BIND_HOST"));
+    }
+    if (getenv("TACHYON_MASTER_WEB_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.MASTER_WEB_BIND_HOST,
+          getenv("TACHYON_MASTER_WEB_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_BIND_HOST, getenv("TACHYON_WORKER_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_DATA_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_DATA_BIND_HOST,
+          getenv("TACHYON_WORKER_DATA_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_WEB_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_WEB_BIND_HOST,
+          getenv("TACHYON_WORKER_WEB_BIND_HOST"));
+    }
+
```

Comment:
User can specify environment variables at either tachyon-env.sh or from OS configuration (e.g. .bashrc on CentOS). For example, user can use "export TACHYON_WORKER_DATA_BIND_HOST=`hostname -A | cut -d" " -f2`" to bind 2nd network interface for running worker data service, and use "export TACHYON_WORKER_BIND_HOST=`hostname -A | cut -d" " -f1`" to bind 1st network interface to run Worker RPC service.